liquibase-core/src/main/java/liquibase/command/core/InternalHistoryCommandStep.java

```diff
@@ -132,12 +164,74 @@ void printReport(PrintWriter output) {
             output.println("");
         }
 
-        String getDeploymentId() {
-            if (changeSets.size() == 0) {
-                return null;
+        @Override
+        public String getDeploymentId() {
+            return changeSets.stream()
+                    .findFirst()
+                    .map(RanChangeSet::getDeploymentId)
+                    .orElse(null);
+        }
+    }
+
+    public static class TabularDeploymentDetails implements ReportPrinter {
+
+        private static final List<String> HEADERS = Arrays.asList(
+                "Deployment ID",
+                "Update date",
+                "Change log path",
+                "Change set author",
+                "Change set ID",
+                "Checksum");
```

Comment:
Actually, `Checksum` is not part of the `LEGACY` output. Is there any specific use case for including it? If there is not, to keep consistency with the current configuration I would recommend not including it.

Reply:
Fixed in 008944bb90519291a595a9fd0bee48000e530e74